steps/mentor-calibre-drc/configure.yml

```diff
@@ -24,6 +24,7 @@ outputs:
 #-------------------------------------------------------------------------
 
 commands:
+  - if [ $drc_env_setup != 'undefined' ]; then source inputs/adk/{drc_env_setup}; fi
```

Comment:
Does this work? The `$drc_env_setup` variable should maybe also be `{drc_env_setup}`?